Source/ActionController.swift

```diff
@@ -364,8 +376,13 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
                 onConfigureSectionHeader?(reusableview!,  sectionForIndex(actionSectionIndexFor((indexPath as NSIndexPath).section))!.data!)
                 return reusableview!
             }
+        } else if kind == UICollectionElementKindSectionFooter {
+            let footerView: CancelCell! = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: ReusableViewIds.SectionFooter.rawValue, for: indexPath) as? CancelCell
```

Comment:
As this class is used as the collection view's footer view, there is no need that it be a `UICollectionViewCell`, instead you can make it to inherit from `UICollectionReusableView`, in such case the name can be changed to just CancelView. Thoughts?